tests/acceptance/README.md

```diff
@@ -0,0 +1,36 @@
+# Acceptance Tests
+
+The following BATs tests can be used to test basic functionality of the GCP Auth Engine.
+
+## Prerequisites
+
+* Clone this repository to your workstation
+* [Bats Core installed](https://bats-core.readthedocs.io/en/stable/installation.html#homebrew)
+* Docker
+* Vault CLI installed
+* GCP Project that has a service account w/ [required permissions](https://www.vaultproject.io/docs/auth/gcp#required-gcp-permissions)
+
+### GCP Testing
+
+First, set the following env variables from your GCP project 
+
+* SERVICE_ACCOUNT_ID
+* PATH_TO_CREDS env variable pointing to service account credentials JSON file
+* GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
Are these the same thing?

Reply:
I don't see `GOOGLE_APPLICATION_CREDENTIALS` being used anywhere

Reply:
this is the path to your application creds.json file. It's a variable used by the Go SDK https://www.vaultproject.io/docs/auth/gcp#authentication

Reply:
Oh I see what you mean, yes this can be the same as `PATH_TO_CREDS`

Reply:
Do they both need to be set?

Reply:
If they do both need to be set, it might be simpler if we just use one. Maybe `GOOGLE_APPLICATION_CREDENTIALS` since that seems to be required anyway.